src/stores/UsersListStore.ts

```diff
@@ -0,0 +1,26 @@
+import { inject, injectable } from "inversify";
+import { makeAutoObservable } from "mobx";
+import ownTypes from "../ioc/ownTypes";
+import User from "../models/User";
+import UsersService  from "../services/UsersService";
+
+@injectable()
+export default class UsersListStore {
+    public users: User[] = [];
+
+    public constructor(
+        @inject(ownTypes.usersService) private readonly usersService: UsersService
+    ) {
+        makeAutoObservable(this);
+    }
+
+    public init = async(): Promise<void> => {
+        try {
+           this.users = await this.usersService.getAllUsers();
+        } catch (e) {
+            if (e instanceof Error) {
+                console.log(e.message);
+            }
```

Comment:
And you will check in every catch construction instance of Error? Maybe it's better to create a method od just remove the check.

Reply:
In addition it's not a best practice to write logs in the code. You should think how to handle errors using another ideas.